src/env.ts

```diff
@@ -1,10 +1,11 @@
-import dotenv from 'dotenv'
+import { LogLevel } from '@credo-ts/core'
+import * as dotenv from 'dotenv'
 import * as envalid from 'envalid'
 import { makeValidator } from 'envalid'
 import { singleton } from 'tsyringe'
 
 if (process.env.NODE_ENV === 'test') {
-  dotenv.config({ path: 'tests/test.env' }) //different configs for unit and integration??
+  dotenv.config({ path: 'tests/test.env.ts' })
```

Comment:
Sorry this seems to have been lost in translation. Just create the `test/test.env` file with the contents

```
LOG_LEVEL=3
```